qiskit/optimization/converters/linear_equality_to_penalty.py

```diff
@@ -31,12 +35,14 @@ def __init__(self):
         self._src = None
```

Comment:
@a-matsuo the parameters should go to the `__init__` the `encode()` should only take the `QuadraticProgram`. The setting should be a property of the converter, such that we can change it not only at the time the `encode()` is used (which is too late, since we usually don't have access there if it is called within an `OptimizationAlgorithm`).